pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[project]
+name = "HASH"
+
+[tool.rye]
+virtual = true
+
+[tool.rye.workspace]
+members = ["apps/lm-prompt-optimization"]
```

Comment:
Do we need to specify them manually or is detection possible?

Reply:
automatic detection is possible, but was a bit tucked away in the docs, fixed in [8a26c92](https://github.com/hashintel/hash/pull/4568/commits/8a26c92f8b1e0baa99f3f587516b91faa4d806c2)